> ```
> 2024-06-18 00:41:53 ++ python type_checking.py
> 2024-06-18 00:41:53 ----------------Check results--------------------
> 2024-06-18 00:41:53 paddle.optimizer.Adamax:1
> 2024-06-18 00:41:53 <string>:11:15: error: Argument "beta1" to "Adamax" has incompatible type "Tensor"; expected "float"  [arg-type]
> 2024-06-18 00:41:53 <string>:12:15: error: Argument "beta2" to "Adamax" has incompatible type "Tensor"; expected "float"  [arg-type]
> 2024-06-18 00:41:53 Found 2 errors in 1 file (checked 1 source file)
> 2024-06-18 00:41:53 
> 2024-06-18 00:41:53 
> 2024-06-18 00:41:53 >>> Mistakes found in type checking!
> 2024-06-18 00:41:53 >>> Please recheck the type annotations. Run `tools/type_checking.py` to check the typing issues:
> 2024-06-18 00:41:53 > python tools/type_checking.py paddle.optimizer.Adamax
> 2024-06-18 00:41:53 ----------------End of the Check--------------------
> ```

adamax 的 beta1、beta2 支持 Tensor？

--- a/python/paddle/optimizer/adamax.py
+++ b/python/paddle/optimizer/adamax.py
@@ -63,9 +63,11 @@ class Adamax(Optimizer):
     Args:
         learning_rate (float|LRScheduler, optional): The learning rate used to update ``Parameter``.
             It can be a float value or a LRScheduler. The default value is 0.001.
-        beta1 (float, optional): The exponential decay rate for the 1st moment estimates.
+        beta1 (float|Tensor, optional): The exponential decay rate for the 1st moment estimates.
+            It should be a float number or a 0-D Tensor with shape [] and data type as float32.
             The default value is 0.9.
-        beta2 (float, optional): The exponential decay rate for the 2nd moment estimates.
+        beta2 (float|Tensor, optional): The exponential decay rate for the 2nd moment estimates.
+            It should be a float number or a 0-D Tensor with shape [] and data type as float32.
             The default value is 0.999.
         epsilon (float, optional): A small float value for numerical stability.
             The default value is 1e-08.
@@ -150,8 +152,8 @@ class Adamax(Optimizer):
     def __init__(
         self,
         learning_rate: float | LRScheduler = 0.001,
-        beta1: float = 0.9,
-        beta2: float = 0.999,
+        beta1: float | Tensor = 0.9,
+        beta2: float | Tensor = 0.999,
         epsilon: float = 1e-8,
         parameters: Sequence[Tensor]
         | Sequence[_AdamParameterConfig]

